content/en/docs/tutorials/kubernetes-basics/_index.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Learn Kubernetes Basics
+main_menu: true
+no_list: true
+weight: 20
+content_type: concept
+card:
+  name: tutorials
+  weight: 20
+  title: Walkthrough the basics
+---
+
+## {{% heading "objectives" %}}
+
+This tutorial provides a walkthrough of the basics of the Kubernetes cluster orchestration
+system. Each module contains some background information on major Kubernetes features
+and concepts, and a tutorial for you to follow along.
+
+Using the tutorials, you can learn to:
+
+* Deploy a containerized application on a cluster.
+* Scale the deployment.
+* Update the containerized application with a new software version.
+* Debug the containerized application.
+
+## What can Kubernetes do for you?
+
+With modern web services, users expect applications to be available 24/7, and developers
+expect to deploy new versions of those applications several times a day. Containerization
+helps package software to serve these goals, enabling applications to be released and updated
+without downtime. Kubernetes helps you make sure those containerized applications run where
+and when you want, and helps them find the resources and tools they need to work. Kubernetes
+is a production-ready, open source platform designed with Google's accumulated experience in
+container orchestration, combined with best-of-breed ideas from the community.
+
+## Kubernetes Basics Modules
+
+<!-- this css code is to preserve the original format -->
+<div style="display: flex; flex-wrap: wrap; gap: 20px; justify-content: space-between;">
+  <div style="width: 30%; text-align: center;">
```

Comment:
Ideally, use CSS classes rather than inline CSS.

Reply:
I moved the classes to a separate css file.